package.json

```diff
@@ -14,7 +14,7 @@
     "url": "https://github.com/facebook/Docusaurus.git"
   },
   "scripts": {
-    "ci-check": "yarn prettier:diff",
+    "ci-check": "yarn prettier && yarn prettier:diff",
```

Comment:
@InternetExplorer7 This should not be added here. `yarn ci-check` is run in Circle this change makes the code prettified before diffing, so it is guaranteed to pass. 

The check for whether the code has been prettified is essentially rendered ineffective by this change as even if submitted code has been prettified, this command will fix it and pass the CI but the changes are not committed to the branch. 

I'll submit a PR to revert this line. cc @JoelMarcey